client/main/analyses.ts

```diff
@@ -408,8 +412,8 @@ const Analyses = Backbone.Model.extend({
     _notifyResultsChanged(analysis) {
         this.trigger('analysisResultsChanged', analysis);
     },
-    _notifyOptionsChanged(analysis) {
-        this.trigger('analysisOptionsChanged', analysis);
+    _notifyOptionsChanged(analysis, incoming) {
+        this.trigger('analysisOptionsChanged', analysis, incoming);
```

Comment:
maybe add a comment explaining what is meant by 'incoming'

Reply:
This is not saying outdated but i have updated it to include a comment. Not sure why.